### PR TITLE
add new play method

### PR DESCRIPTION
**Pull request by nuwanif**

### CODE REVIEW

**Review by supunwathsana — COMMENTED** [review]
Ok with changes

File: Robot/src/Follow.java [hunk 1]
```diff
@@ -158,4 +158,9 @@ public void play() {
 		Sound.playSample(new File("detected.wav"), Sound.VOL_MAX);
 		Sound.playSample(new File("stop.wav"), Sound.VOL_MAX);
 	}
+	public void play2() {
```
Comment (dinusha4010) [inline Robot/src/Follow.java:161]: Please remove Sound.playSample(new File("stop.wav"), Sound.VOL_MAX);

File: Robot/src/Follow.java [hunk 2]
```diff
@@ -158,4 +158,9 @@ public void play() {
...
+		Sound.playSample(new File("object.wav"), Sound.VOL_MAX);
+		Sound.playSample(new File("detected.wav"), Sound.VOL_MAX);
```
Comment (dinusha4010) [inline Robot/src/Follow.java:163]:
```suggestion
		Sound.playSample(new File("detected.wav"), Sound.VOL_MAX);
		Sound.playSample(new File("detected.wav"), Sound.VOL_MAX);
```

**Review by dinusha4010 — CHANGES_REQUESTED** [review]
play detected.wav twice

**Comment by nuwanif** [comment]
changes done

**Review by dinusha4010 — APPROVED** [review]

**Review by supunwathsana — COMMENTED** [review]
correct

**Review by supunwathsana — COMMENTED** [review]
correct

**Review by dinusha4010 — APPROVED** [review]
changed accordingly

**Review by nuwanif — COMMENTED** [review]
approve

**Review by supunwathsana — APPROVED** [review]
Correct

**Pull request merged**